### PR TITLE
data: 1js2002

### PR DESCRIPTION
**Pull request by 1js2002**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5894"></a>

### CODE REVIEW

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @1js2002, thank you for contributing :tada: 
Please rename your file and add **`data/`** in the beginning :+1: 
It will look like **`data/1js2002.json`**

**Review by 1js2002 — COMMENTED** [review]
changed the filename from "1js2002.json" to "data/1js2002.json"

File: data/1js2002.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "bio": "MERN stack developer",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/1js2002",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://github.com/1js2002",
```
Comment (kumarsonsoff3) [inline data/1js2002.json:13]: Did you mean with **GitHub** here!?

```suggestion
      "name": "Follow me on GitHub",
      "url": "https://github.com/1js2002",
```

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks awesome. Great collaboration everyone. 👍

**Pull request merged**